packages/pds-ember/app/styles/pds/tokens/_font.scss

```diff
@@ -4,8 +4,24 @@ $_module: "PDS.Tokens.Font";
 
 // ------ FONT FAMILIES ------ //
 
+// TODO: clean up
 // "system" monospace font stack
-$family--mono: monospace;
+$family--mono:
+  "SF Mono",
+  "SFMono-Regular",
+  "Consolas",
+  "Liberation Mono",
+  "Menlo",
+  "Courier New", Courier,
+  monospace;
+/*
+$family--mono:
+  "SFMono-Regular",
+  "Consolas",
+  "Liberation Mono",
+  "Menlo",
+  monospace;
+*/
```

Comment:
- [ ] finish updates to the monospace font stack

FYI: this is needed because `monospace` is likely to load a font that doesn't have adequate font-size, line-height, and font-weight customization, which results in a jarring experience when sans-serif and monospace text is side-by-side.